edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/DependencyManager.cs

```diff
@@ -154,6 +154,7 @@ void RegisterMqttModule(ContainerBuilder builder, (bool isEnabled, bool usePersi
             int configUpdateFrequencySecs = this.configuration.GetValue("ConfigRefreshFrequencySecs", 3600);
             TimeSpan configUpdateFrequency = TimeSpan.FromSeconds(configUpdateFrequencySecs);
             bool checkEntireQueueOnCleanup = this.configuration.GetValue("CheckEntireQueueOnCleanup", false);
+            bool closeCloudConnectionWhenCloseDeviceConnection = this.configuration.GetValue("CloseCloudConnectionWhenCloseDeviceConnection", true);
```

Comment:
Looks like this comment was not addressed - 
https://github.com/Azure/iotedge/pull/3100#discussion_r440533195
Can you pls fix that and cherry pick that as well?

Reply:
Fixed in #3109 and once it is merged, then will cherry-pick to this PR.